examples/amp-subscriptions.amp.html

```diff
@@ -207,9 +207,10 @@ <h1 itemprop="headline">Lorem Ipsum</h1>
           </div>
         </div>
 
-        <section subscriptions-section="actions" class="login-section">
-          <div subscriptions-action="login" subscriptions-display="NOT loggedIn">Login</div>
-        </section>
+
+        <div subscriptions-action="login"
+          subscriptions-display="NOT loggedIn"
+          subscriptions-decorator="subscribe.google.com">Login</div>
```

Comment:
Let's discuss, but I'd prefer this to be done this way:

```
<button subscriptions-action="login"
  subscriptions-service="subscribe.google.com"
  subscriptions-decorate>
</button>
```

Or something like this. I think the "decoration" is not a strong enough word to indicate execution with a certain service. It's also optional.

Reply:
So it looks like it's actually renamed to `subscriptions-service` in the code already. Let's just update the example then.

Reply:
Added the button